test/common/webdriver_bidi.py

```diff
@@ -187,7 +187,9 @@ async def close(self) -> None:
 
         await self.ws.close()
         await self.close_bidi_session()
-        self.bidi_session.process.terminate()
+        # Terminate, but only if still running
+        if self.bidi_session.process.returncode is None:
+            self.bidi_session.process.terminate()
```

Comment:
That feels like a TOCTOU? Safer to do try/except: and catch a ProcessLookupError? Or is that somehow guaranteed to work?

Reply:
No, you are right.